src/libraries/System.Text.Json/tests/Common/ReferenceHandlerTests/ReferenceHandlerTests.Serialize.cs

```diff
@@ -3,17 +3,18 @@
 
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Threading.Tasks;
```

Comment:
Guessing the build failures are related to missing Newtonsoft references in the SourceGen test project?